src/components/Upgrade/Navigation.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { ButtonBase, springs } from '@aragon/ui'
+import { ButtonBase, IconRight, IconLeft, springs } from '@aragon/ui'
 import { useSpring, animated } from 'react-spring/hooks'
-import arrowSvg from './assets/arrow.svg'
```

Comment:
Should we remove this file?

Reply:
Yes! We can, seems it's not used anywhere else.